pyCGM_Single/pyCGM.py

```diff
@@ -38,39 +38,38 @@ def pelvisJointCenter(frame):
     Calculates the pelvis joint center and axis and returns both.
     -------------------------------------------------------------------------
     
-    INPUT: 
-        dictionaries of marker lists.  
+    Parameters
+    ---------- 
+    frame : dict 
+        Dictionaries of marker lists.
             { [], [], [] }
     
-    OUTPUT: Returns the origin and pelvis axis also sacrum
+    Returns
+    -------
+    array
+        Returns the origin and pelvis axis also sacrum. 
             Pelvis = [[origin x,y,z position],
                       [[pelvis x_axis x,y,z position],
                        [pelvis y_axis x,y,z position],
                        [pelvis z_axis x,y,z position]],
                        [sacrum x,y,z position]]    
-    MODIFIES: -
-    
-    -------------------------------------------------------------------------
     
-    EXAMPLE:
-            i = 3883
-            frame = {...,'LASIX': 183.18504333, 'LASIY': 422.78927612, 'LASIZ': 1033.07299805,  
-                            'LPSIX': 255.79994202, 'LPSIY': 241.42199707, 'LPSIZ': 1057.30065918,        
-                            'RASIX': 395.36532593, 'RASIY': 428.09790039, 'RASIZ': 1036.82763672,
-                            'RPSIX': 341.41815186, 'RPSIY': 246.72117615, 'RPSIZ': 1055.99145508}
-
-            pelvisJointCenter(frame)
-
-            >>> [array([ 289.27518463, 425.44358826, 1034.95031738]),
-                array([[ 289.25243803, 426.43632163, 1034.8321521],
-                    [ 288.27565385, 425.41858059, 1034.93263017],
-                    [ 289.25467091, 425.56129577, 1035.94315379]]),
-                array([ 298,60904694, 244.07158661, 1056.64605715])]
-
-
+    Modifies
+    ---------
+    -
+
+    Example
+    -------
+    >>> frame = {'RASI': np.array([ 395.36532593,  428.09790039, 1036.82763672]), 
```

Comment:
Please import numpy and the root level pycgm directory into this file to make it a full working example. It will come up later. 

Also, it would be good to put some of the comments in the description as well. For example, the comments explain the calculation method and the required markers to be passed in. Mentioning these things will be beneficial to end users (and when we build the docs)

Reply:
I have added the imports to all the doctests that needed them, and have added additional helpful comments to the docstrings.